src/game/common/thing/thingfactory.cpp

```diff
@@ -68,12 +66,12 @@ ThingTemplate *ThingFactory::New_Override(ThingTemplate *thing_template)
         "Thing template '%s' not in master list",
         thing_template->Get_Name().Str());
 
+    ThingTemplate *override_template = static_cast<ThingTemplate *>(thing_template->Friend_Get_Final_Override());

```

Comment:
Looks like it was `Friend_Get_Final_Override`, which is same as `Get_Final_Override` though. But it allows for `static_cast` here, which is nicer.